angularjs-portal-home/src/main/webapp/my-app/layout/widget/partials/widget-card.html

```diff
@@ -107,7 +109,7 @@
             <portlet-icon></portlet-icon>
           </div>
         </a>
-        <button aria-labelledby="goToApps-{{portlet.nodeId}} appTitle_portlet.title-{{portlet.nodeId}}" class="launch-app-button"  ng-click="widgetCtrl.maxStaticPortlet(portlet)">
+        <button aria-label="{{ portlet.widgetConfig.launchText ? portlet.widgetConfig.launchText : 'Launch full app' }}" class="launch-app-button"  ng-click="widgetCtrl.maxStaticPortlet(portlet)">
```

Comment:
I attempted to address this by referring to the widgetConfig, putting the responsibility on the widget creators, rather than on our widget templates. I think that's the best option from a separation-of-responsibilities perspective (i.e. If we're going to offer the ability to set aria-labels in the widgetConfig, we should encourage the use of that feature). 

I made my initial decision based on our [launch text guidance](http://uw-madison-doit.github.io/angularjs-portal/latest/#/md/widget-launch-button), but I think Jess would agree that a more meaningful fallback for aria-labels would be a good thing. Any objections to defaulting to `Launch {{ portlet.title }}`?